Model/GraphQlReader.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Graycore\CustomAttributeMetadataCache\Model;
+
+use Magento\Framework\Config\ReaderInterface;
+
+class GraphQlReader implements ReaderInterface
+{
+    public function read($scope = null)
```

Comment:
danslo — Today at 1:37 PM
the graphql reader is just so that I don't have to copy stuff from core and then only add a cache identity. that's mess with other potential modifications to this query (maybe someone else adding a parameter or changing the return type of the query, etc)
You're free to change it to use schema.graphqls, the "hard" stuff is in the CacheIdentity class 😛
and I pushed a change for the unit test
Candle — Today at 1:41 PM
Ah, ok. I like your version.